src/components/NFTPortfolio/Item.vue

```diff
@@ -102,6 +102,7 @@ export default {
   methods: {
     fetchInfo() {
       this.lazyFetchNFTClassAggregatedData();
+      this.updateNFTOwners();
```

Comment:
Or we should allow `lazyFetchNFTClassAggregatedData` to ignore some fields by passing parameter
- Ideally we should ignore fields by passing `data` qs to API, since the API already support this
- If we are too lazy to figure out what qs to pass to API, we can still selectively not commit data back to store according to parameters